File: sdk/samples/WPFSamples/CodeOnlyWindowsApplicationSample/visualbasic/entrypoint.vb
Imports System

Namespace CodeOnlyWindowsApplicationSample
    Public Class EntryPoint

        ' Entry point function

        ' WPF applications must run in a single-threaded apartment
        <STAThread()> _
        Public Shared Sub Main()
            ' Start the WPF application
            Dim _app As New App
            _app.Run()
        End Sub

    End Class
End Namespace

File: sdk/samples/WPFSamples/CodeOnlyWindowsApplicationSample/csharp/entrypoint.cs
using System;

namespace CodeOnlyWindowsApplicationSample
{
    public class EntryPoint
    {
        // Entry point function

        // WPF applications must run in a single-threaded apartment
        [STAThread]
        public static void Main()
        {
            // Start the WPF application
            App app = new App();
            app.Run();
        }
    }
}
